src/ensembl/src/shared/components/help-popup/HelpPopupButton.tsx

```diff
@@ -28,7 +29,16 @@ import { SlugReference } from './types';
 
 import styles from './HelpPopupButton.scss';
 
-type Props = SlugReference;
+type ClassNames = Partial<{
+  label: string;
+  button: string;
+}>;
+
+type Props = {
+  classNames?: ClassNames;
+  text: string;
+  slug: SlugReference;
```

Comment:
What's the benefit of changing the field type from `slug: string` to `slug: { slug: string }`?

Reply:
Not sure if there is a benefit. I was thinking that maybe we will need to extend `SlugReference` to pass other details.

Reply:
Let's change it if we discover that we need to, not before 🙂